resources/js/components/pages/auth/Login.vue

```diff
@@ -119,6 +127,31 @@ const checkVerified = () => {
     }
 };
 
+const resendVerification = async () => {
+    try {
+        if (!email.value) {
+            snackbar.error({
+                title: 'Email required',
+                text: 'Please enter your email address.',
+            });
+
+            return;
+        }
+        await AuthApi.resendVerificationEmail(email.value);
+        snackbar.success({
+            title: 'Verification email sent.',
+            text: 'Please check your inbox.',
+            save: false,
+        });
+    } catch (e) {
+        if (snackbarErrors(e)) return;
+        snackbar.error({
+            title: 'An error occurred while sending the verification email.',
+            text: 'Please try again.',
```

Comment:
Should we catch the error related to too many attempts, rather than displaying a generic error. That will be very informative

Reply:
the `snackbarErrors` will catch the errors coming from the platform, in case of none of the issue is found/ returned it will show the generic error message